internal/dag/dag.go

```diff
@@ -361,7 +360,7 @@ type SecureVirtualHost struct {
 	VirtualHost
 
 	// TLS minimum protocol version. Defaults to envoy_api_v2_auth.TlsParameters_TLS_AUTO
-	MinTLSVersion envoy_api_v2_auth.TlsParameters_TlsProtocol
+	MinTLSVersion string
```

Comment:
I think this is fine and matches other types. The only thing we could think about is making this an enum then using that later on such that we wouldn't need to keep parsing strings, but could always be a later refactor.